manager/manager_test.go

```diff
@@ -400,3 +400,110 @@ func TestManagerLockUnlock(t *testing.T) {
 	// error.
 	<-done
 }
+
+// If the root CA materia is updated in the memory store, a manager will update its own
```

Comment:
Looks like it wouldn't have caught this one anyway, I guess because `materia` is a real word, although [latin](http://www.dictionary.com/browse/materia-medica)).